src/main/scala/com/lunatech/lunabot/model/FromInfo.scala

```diff
@@ -0,0 +1,8 @@
+package com.lunatech.lunabot.model
+
+/**
+ * Created by dimitrioscharoulis on 12/10/15.
```

Comment:
No need for this comment  - the name of the author and the date are in git anyway, Instead add more explanative comment why this class is here.